index.js

```diff
@@ -0,0 +1,199 @@
+import { arrSliderText } from "./sliderText.js";
+const containerHeaderMenu = document.getElementsByClassName(
+  "containerHeaderMenu"
+)[0];
+const gamburgerContainer =
+  document.getElementsByClassName("gamburgerContainer")[0];
+const gamburger = document.getElementsByClassName("gamburger")[0];
```

Comment:
type in constant name, also you have only one hamburger element use id instead

Reply:
done